be/src/storage/rowset/bitmap_index_writer.cpp

```diff
@@ -40,14 +40,64 @@ namespace starrocks {
 
 namespace {
 
+static const size_t roaring_get_size_fastpath_threshold = 1024;
+
+class RoaringWrapper {
+public:
+    explicit RoaringWrapper(rowid_t rid)
+            : _roaring(std::move(Roaring::bitmapOf(1, rid))),
+              _old_size(1),
+              _size_changed(true),
+              _enable_get_size_in_Bytes(false){};
+    Roaring& roaring() { return _roaring; }
+    uint64_t roaring_size() const { return _roaring.getSizeInBytes(false); }
+    uint64_t old_size() const { return _old_size; }
+    void set_old_size(uint64_t size) const { _old_size = size; }
+    bool size_changed() const { return _size_changed; }
+    void set_size_changed() const { _size_changed = true; }
+    void unset_size_changed() const { _size_changed = false; }
+
+    void add_rid(const rowid_t rid, uint64_t& reverted_index_size, vector<RoaringWrapper*>& size_changed_roaring_vec) {
+        _roaring.add(rid);
+        if (_enable_get_size_in_Bytes) {
```

Comment:
When does the _enable_get_size_in_Bytes set to be true?

Reply:
When the _element_count is less than roaring_get_size_fastpath_threshold, we will estimate the roaring bitmap size by ourselves.
When the _element_count is larger than roaring_get_size_fastpath_threshold, the `_enable_get_size_in_Bytes` is set,  and we will use the roaring bitmap's `getSizeInByte` function to obtain the exact size.